### ISSUE TITLE
Review usage of multiple yaml config files

### ISSUE THREAD

**Issue by btat** [SUSE/doc-cap#678]
Based on feedback from investigation for CAP-1206

```
Any changes made to a running cluster should be done by updating a single config file. 
Otherwise there is a big risk that they will get reverted on the next upgrade (so this 
might be an issue for other parts of the docs too).
```

We should review the docs for usage of multiple yaml config files and check for instances where the config changes are not one time occurrences and need to be persisted for the life of the cluster.

A specific example of this is CCDB Secret Rotation (CAP-1206) where we suggest placing new keys into new-key-values.yaml and not into scf-config-values.yaml. These new settings will have to be included in all future helm upgrade commands; otherwise the CC will no longer be able to decrypt values using the new key (and revert to using the old key for encrypting any new secrets).

**Comment by btat** [comment]
Example instance addressed via https://github.com/SUSE/doc-cap/pull/696

**Comment by btat** [comment]
Closing as most/all of these should have been fixed during the 2.0 release. If there are any remaining, a new issue can be opened for the specific issue instead.
